workflow/state-machines.rst

```diff
@@ -209,6 +209,33 @@ you can get this state machine by injecting the Workflow registry service::
         public function someMethod($subject)
         {
             $stateMachine = $this->workflows->get($subject, 'pull_request');
+            $stateMachine->apply($subject, 'wait_for_review');
+            // ...
+        }
+
+        // ...
+    }
+
+Symfony also creates automatically for you a service for each workflow (:class:`Symfony\\Component\\Workflow\\Workflow`) or state machine (:class:`Symfony\\Component\\Workflow\\StateMachine`) you have defined in your configuration.
```

Comment:
I would reword it:

Symfony creates a service for you automatically....

Reply:
see e1e4efcd707cfa7489d6df4ba47a6d1be4a308ea :)

Reply:
Thanks